server/src/test/java/PerkTest.java

```diff
@@ -1,8 +1,24 @@
 import static org.junit.jupiter.api.Assertions.*;
 import org.junit.jupiter.api.Test;
-
+import com.example.model.Perk;
 import java.util.ArrayList;
 
 class PerkTest {
 
+    @Test
+    public void PerkTest(){
+        Perk perk = new Perk();
```

Comment:
Make sure to test all properties such as the Id, upvote, downvotes, etc. so you can test the methods